src/i18n/fr/nav.ts

```diff
@@ -21,32 +25,33 @@ export default NavDictionary({
 	'core-concepts/astro-components': 'Composants',
 	'core-concepts/astro-pages': 'Pages',
 	'core-concepts/layouts': 'Layouts',
-	'guides/markdown-content': 'Markdown',
+	'guides/markdown-content': 'Contenu Markdown',
 	'guides/imports': 'Fichiers Statiques',
 	'guides/troubleshooting': 'Dépannage',
 
 	// Features
 	features: 'Fonctionnalités',
 	'guides/configuring-astro': 'Configurer Astro',
-	'guides/styling': 'CSS et Styles',
-	'guides/data-fetching': 'Récupération de Données',
+	'guides/styling': 'CSS et Style',
+	'guides/data-fetching': 'Récupération de données',
```

Comment:
I’m noticing that many of the entries use title case, so maybe we should stick with capital D?

```suggestion
	'guides/data-fetching': 'Récupération de Données',
```